aws/extensions/computetask_awswin_hamletenv/extension.ftl

```diff
@@ -66,6 +66,11 @@
         priority=2
         engine=AWS_EC2_CFN_INIT_COMPUTE_TASK_CONFIG_TYPE
         content={
+                "packages" : {
+                    "msi" : {
+                        "awscli": "https://awscli.amazonaws.com/AWSCLIV2.msi"
+                    }
```

Comment:
Should we move this to the awscli step? then its handled as part of the same extension?

Reply:
Good point - done